server/tests/metadata.rs

```diff
@@ -103,4 +103,128 @@ mod entities_metadata_query {
             .await
             .should_be_bad_request();
     }
+
+    #[actix_rt::test]
+    async fn checks_mandatory_keys_in_response() {
+        Message::new("EntitiesMetadataQuery", json!({ "first": 350 }))
+            .execute()
+            .await
+            .should_be_ok_with(|value| {
+                let mandatory_keys = [
+                    "@context",
+                    "id",
+                    "type",
+                    "name",
+                    "publisher",
+                    "learningResourceType",
+                ];
+
+                let entities = value["entities"].as_array().unwrap();
+                for entity in entities {
+                    for mandatory_key in &mandatory_keys {
+                        assert!(
+                            entity.get(mandatory_key).is_some(),
+                            "Mandatory key '{}' is missing from the response",
+                            mandatory_key
+                        );
+                    }
+                }
+            });
+    }
+
+    #[actix_rt::test]
+    async fn checks_mandatory_keys_in_response_of_a_quiz() {
+        // Fetching a quiz https://serlo.org/20256
+        Message::new(
+            "EntitiesMetadataQuery",
+            json!({ "first": 1, "after": 20_000  }),
+        )
+        .execute()
+        .await
+        .should_be_ok_with(|value| {
+            assert_eq!(
+                value["entities"][0]["name"],
+                "Quiz: https://serlo.org/20256"
+            );
+            let mandatory_keys = [
+                "@context",
+                "id",
+                "type",
+                "name",
+                "publisher",
+                "learningResourceType",
+            ];
+
+            let entities = value["entities"].as_array().unwrap();
+            for entity in entities {
+                for mandatory_key in &mandatory_keys {
+                    assert!(
+                        entity.get(mandatory_key).is_some(),
+                        "Mandatory key '{}' is missing from the Quiz!",
+                        mandatory_key
+                    );
+                }
+            }
+        });
+    }
```

Comment:
```suggestion
```

When there is at least one exercise in the first 350 entities there is no additional value of this test than just having `default_value_for_property_name()` and `checks_mandatory_keys_in_response()`